walletd/renderer/components/useConfigData.tsx

```diff
@@ -16,12 +16,14 @@ export type Config = {
     mode: 'personal' | 'full' | 'none'
     batchSize: number
   }
-  consensus: {
-    network: 'mainnet' | 'zen'
-    gatewayAddress: string
+  syncer: {
+    address: string
     bootstrap: boolean
     enableUPnP: boolean
   }
+  consensus: {
+    network: 'mainnet' | 'zen' | 'anagami'
```

Comment:
Don't enable `anagami` in the UI. It's not a testnet we want most users to use.

Reply:
👍🏼 removed

Reply:
I left the types, but removed the dropdown option.